src/main/java/seedu/address/model/tag/TagStatus.java

```diff
@@ -7,13 +7,48 @@ public enum TagStatus {
     COMPLETE_GOOD, // complete before deadline
     COMPLETE_BAD, // complete after deadline
     INCOMPLETE_GOOD, // incomplete before deadline
-    INCOMPLETE_BAD; // incomplete after deadline
+    INCOMPLETE_BAD, // incomplete after deadline
+
+    PRESENT,
+    ABSENT,
+    ABSENT_WITH_REASON,
+
+    W08, W09, W10, W11, W12, W13,
```

Comment:
I'm a bit confused with the representation here. Why would the code for tutorial sessions be a valid enum for tagStatus?

Reply:
that was the original plan from our dicussion - using enum to limit the valid tutorial slot value.

however in today's meeting, I would propose to have two enum states for TutorialTag instead: {ASSIGNED, AVAILABLE}, with the first to indicate assigned tutorial for both student and tutor, and the latter for the tutor to indicate other slots they are available. Under this implementation scheme, the tutorial slots would be represented by TagName, and the validity of such TagNames (the tutorial slot has to exist to be inserted) can be checked by an additional construct, maybe a set of valid tutorial codes

Reply:
Yup I think that would be a better idea too